livre/eq/render/shaders/fragRayCast.glsl

```diff
@@ -73,9 +74,19 @@ vec3 calcTexturePositionFromAABBPos( vec3 pos )
 // AABB-Ray intersection ( http://prideout.net/blog/?p=64 ).
 bool intersectBox( Ray r, AABB aabb, out float t0, out float t1 )
 {
+    //We need to avoid division by zero in "vec3 invR = 1.0 / r.Dir;"
+    if( r.Dir.x == 0 )
+        r.Dir.x = FLOAT_EPSILON;
```

Comment:
:) You dont need "FLOAT_" there ! I have never seen an integer epsilon so far ( although may be there ! )